_fossfund/0-ooui.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Ooui
-logo: 
+logo: ooui.svg
```

Comment:
There doesn't seem to be an `ooui.svg` present under the `assets` > `images` > `fossfund` directory. I think this, 
https://raw.githubusercontent.com/praeclarum/Ooui/master/Documentation/Icon.png, is the current Ooui logo.